http/http_test.go

```diff
@@ -72,7 +86,7 @@ func testHttpData(t *testing.T, method string, token string, addr string, body i
 	}
 
 	resp, err := client.Do(req)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "checkRedirect disabled for test") {
```

Comment:
Oh, I remember why I did this: the Go http client adds a bunch of extra stuff to the error message. So either way you have to do a string contains check.